packages/site/site-react/src/blocks/DamVideoBlock.tsx

```diff
@@ -40,35 +40,34 @@ export const DamVideoBlock = withPreview(
         }
 
         const [showPreviewImage, setShowPreviewImage] = useState(true);
-        const [isHandledManually, setIsHandledManually] = useState(!autoplay);
+        const [isPlaying, setIsPlaying] = useState(autoplay ?? false);
         const hasPreviewImage = Boolean(previewImage && previewImage.damFile);
 
         const [videoElement, setVideoElement] = useState<HTMLVideoElement | null>(null);
         const videoRef = setVideoElement;
 
         const handleInView = useCallback(
             (inView: boolean) => {
-                if (autoplay && videoElement && !isHandledManually) {
+                if (autoplay && videoElement && isPlaying) {
```

Comment:
Is it correct to use the isPlaying state here? Shouldn't we have both a isHandledManually and a isPlaying state? 🤔